flask/ctx.py

```diff
@@ -321,10 +322,15 @@ def copy(self):
         request object is locked.
 
         .. versionadded:: 0.10
+
+        .. versionchanged:: 1.1
+           A copy of the current session object is added to the request context
```

Comment:
It's not a copy, it's the same object, so this is a bit misleading. Should be "The current session object is used instead of reloading the original cookie from the copied request." Same with the other places you used "copy".

Reply:
Fair point @davidism - fixed the docstrings and the changelog